pyface/ui/qt4/data_view/data_view_item_model.py

```diff
@@ -12,14 +12,15 @@
 
 from pyface.i_image_resource import IImageResource
 from pyface.qt import is_qt5
-from pyface.qt.QtCore import QAbstractItemModel, QModelIndex, Qt
+from pyface.qt.QtCore import QAbstractItemModel, QMimeData, QModelIndex, Qt
 from pyface.qt.QtGui import QColor
 from pyface.data_view.abstract_data_model import AbstractDataModel
 from pyface.data_view.abstract_value_type import CheckState
 from pyface.data_view.data_view_errors import (
     DataViewGetError, DataViewSetError
 )
 from pyface.data_view.index_manager import Root
+from .data_wrapper import DataWrapper
```

Comment:
nit : same comment as earlier about style variations in absolute vs relative imports.

Reply:
Leaving this one here because the convention seems to be that within `pyface.ui.*` you do local imports.

I will open an issue more generally.